onnxruntime/core/providers/cuda/gpu_data_transfer.cc

```diff
@@ -60,4 +64,78 @@ common::Status GPUDataTransfer::CopyTensor(const Tensor& src, Tensor& dst, int e
   return Status::OK();
 }
 
+void PrintTensor(const Tensor& tensor, const std::string& name, bool summary)
```

Comment:
>PrintTensor [](start = 5, length = 11)

This is nice to have more debuggability. Moreover, should we consider a macro to turn it off when not needed, to save some binary size?